code/addons/docs/template/stories/stories-mdx/raw-source.stories.mdx

```diff
@@ -0,0 +1,9 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import * as stories from './raw-source.non-stories';
+import { RawSource } from './RawSource';
+
+<Meta component={RawSource} />
+
+<Canvas>
+  <Story story={stories.RawSource} />
+</Canvas>
```

Comment:
This test case doesn't make sense anymore, since MDX Stories won't be supported in 8.0 anyway.